uat/codebuild/uat_linux_buildspec.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      java: corretto11
+  build:
+    commands:
+      - mvn -U -ntp clean verify -f uat/pom.xml
+      - mvn -U -ntp verify -DskipTests=true
+      - java -Dggc.archive=./target/aws.greengrass.nucleus.zip
+        -Dtags=HelloWorldTst -Dggc.install.root=$CODEBUILD_SRC_DIR -Dggc.log.level=INFO -Daws.region=us-east-1
+        -jar uat/target/nucleus-uat-artifact.jar
```

Comment:
This would not work it needs to be the name of the jar we are creating `greengrass-log-manager-uat-artifact`